contracts/test/ImaMock.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/*
+    ImaMock.sol - SKALE Manager
+    Copyright (C) 2022-Present SKALE Labs
+    @author Vadim Yavorsky
+
+    SKALE Manager is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    SKALE Manager is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with SKALE Manager.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.8.11;
+
+import "@skalenetwork/ima-interfaces/IMessageReceiver.sol";
+
+interface IImaMock {
```

Comment:
Maybe it's better to create `IMessageSender` interface with `postOutgoingMessage` function in the `ima-interfaces` repo, to inherit `IMessageProxy` from it and make `ImaMock` a child of `IMessageSender`?

In current implementation there is no guaranty that `ImaMock` has the same interface as `MessageProxy` does.